dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-athena/src/main/java/org/apache/dolphinscheduler/plugin/datasource/athena/AthenaDataSourceChannelFactory.java

```diff
@@ -32,6 +33,6 @@ public DataSourceChannel create() {
 
     @Override
     public String getName() {
-        return "athena";
+        return DbType.ATHENA.getDescp();
```

Comment:
What about change the descp to `identify` or `name` in DbType, since descp means description, this is easy to be changed.

Reply:
> What about change the descp to `identify` or `name` in DbType, since descp means description, this is easy to be changed.

Change it to this (DbType.MYSQL.name().toLowerCase()), and it shouldn't be changed so easily

Reply:
You can add new field `name` or `identify` in DbType.

Reply:
> You can add new field `name` or `identify` in DbType.

Modifications completed